components/AccountBalance/index.js

```diff
@@ -0,0 +1,43 @@
+import styled from "styled-components";
+
+export default function AccountBalance({ transactions }) {
+  let totalBalance = 0;
+  for (const transaction of transactions) {
+    totalBalance += transaction.amount;
+  }
+  const isNegative = totalBalance < 0;
+
+  return (
+    <StyledAccount >
+      <StyledTotalAmount isNegative={isNegative}>
```

Comment:
```suggestion
      <StyledTotalAmount $isNegative={isNegative}>
```
Please prefix with `$` since it is just for styling and you don't want this prop in the DOM.

Reply:
Done